pkg/integrations/v4/dm/sender_test.go

```diff
@@ -183,12 +183,12 @@ func Test_sender_SendPrometheusSummary(t *testing.T) {
 		telemetry.Summary{
 			Name:       "PrometheusSummaryMetric_sum",
 			Attributes: map[string]interface{}{"att_key": "att_value"},
-			Sum:        float64(2),
+			Sum:        float64(0),
 			Min:        math.NaN(),
 			Max:        math.NaN(),
 			Count:      1,
 			Timestamp:  cannedDate.Add(time.Second),
-			Interval:   cannedDuration,
```

Comment:
Is `cannedDuration` still in use? Why did we do this change?

Reply:
I see, it is still in use `cannedDurationInt := int64(cannedDuration.Seconds() * 1000)`

Reply:
Yes and, from the metric perspective we calculate the sum value from a delta so interval now represent that window instead of what the integration inform us.

More info [here](https://docs.newrelic.com/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/report-metrics-metric-api)